lib/ims/lti/services/tool_consumer_profile_service.rb

```diff
@@ -0,0 +1,16 @@
+module IMS::LTI::Services
+  class ToolConsumerProfileService
+
+    attr_accessor :tcp
+
+    def initialize(tool_consumer_profile)
+      @tcp = tool_consumer_profile
+    end
+
+    def supports_capabilities?(capability, *capabilities)
```

Comment:
If you just use a single splat argument the method takes 0-* arguments.  The way it's no the method take 1-* arguments.